microcosm_flask/audit.py

```diff
@@ -342,7 +367,7 @@ def login(username, password):
             ...
     """
     include_request_body = graph.config.audit.include_request_body
-    include_response_body = graph.config.audit.include_response_body
+    include_response_body = int(graph.config.audit.include_response_body)
```

Comment:
So this probably wouldn't have really worked beore if we would receive it as a string from the ENV or elsewhere?

Reply:
yes. and I need to add the missing `int` above.